rasa/server.py

```diff
@@ -644,6 +644,15 @@ def create_app(
 
     # Setup the Sanic-JWT extension
     if jwt_secret and jwt_method:
+        # `sanic-jwt` depends on having an available event loop when making the call to
+        # `Initialize`. If there is non, the server startup will fail with
+        # `There is no current event loop in thread 'MainThread'`.
+        try:
+            _ = asyncio.get_running_loop()
+        except RuntimeError:
+            new_loop = asyncio.new_event_loop()
```

Comment:
Where does this normally happen? Seems strange we should have to do this ourselves. Maybe we are calling `Initialize` at the wrong time?